docs/development/language-features.md

```diff
@@ -0,0 +1,208 @@
+# Contributing to a Language
+
+Improving Yomitan's features for the language(s) you are interested in is pretty simple, and a great way to help yourself and others. This guide will help you get started.
+
```

Comment:
Is there any linguistic prerequisite people need to have?

Reply:
Nothing in particular that I can think of. Knowing basic parts of speech might help with writing deinflection, but I wanna make this as not scary as possible. If someone does something dumb, we should catch it in review anyway.